src/components/sheet/SimpleSheet.tsx

```diff
@@ -32,6 +34,7 @@ export const SimpleSheet = ({
       removeTopPadding
       scrollEnabled={scrollEnabled}
       backgroundColor={backgroundColor}
+      onDismiss={onDismiss}
```

Comment:
this is already set up i image?

Reply:
yeah, it's a prop of `SlackSheet`. i confirmed that it gets called when the sheet dismisses, whether it is dismissed by gesture or by button press